source/transfer_list.rst

```diff
@@ -245,10 +245,11 @@ Anyone is free to use tags from that range for any custom TE layout without
 adding their definitions to this specification first. The use of this range is
 *strongly discouraged* for anything other than local experiments or code that
```

Comment:
"The use of this range should not be used...".

Reply:
Done